### DISCUSSION TITLE
ESP32-A1S Mute detection

### DISCUSSION THREAD

**Discussion by SandeepJagdev** [pschatzmann/arduino-audio-tools#994] [Q&A] [answered]
I am using ESP32-A1S module with TPA3116 ic. Everything works fine. Now i want to control Mute and shut down pins of TPA3116 with ESP32 . i want to shut down tpa3116  when no audio is played via bluetooth and also via line . Please help.

**Comment by pschatzmann** [comment]
Where exactly is your problem ? 

I suggest to use the VolumeOutput class to determine if you have any audio.

**Reply by SandeepJagdev** [reply]
Hello sir , I tried from last two  days but not works. Please help. any short example

**Comment by pschatzmann** [comment, marked as answer]
What is not working ?
Did you have a look at the existing example ? https://github.com/pschatzmann/arduino-audio-tools/blob/main/examples/examples-stream/streams-generator-volume/streams-generator-volume.ino
